pkg/testhelper/testhelper.go

```diff
@@ -113,6 +113,8 @@ const (
 	DeploymentName               = "Deployment Name"
 	StatefulSetName              = "StatefulSet Name"
 	PodDisruptionBudgetReference = "Pod Disruption Budget Reference"
+	CrdName                      = "Crd Name"
```

Comment:
I think the strings should be `CRD` instead of `Crd` just as far as the acronym goes.

Reply:
Done.